core/src/main/scripts/importer/cbioportal_common.py

```diff
@@ -222,7 +234,8 @@ class MetaFileTypes(object):
     "org.mskcc.cbio.portal.scripts.ImportGisticData" : False,
     "org.mskcc.cbio.portal.scripts.ImportMutSigData" : False,
     "org.mskcc.cbio.portal.scripts.ImportProfileData" : True,
-    "org.mskcc.cbio.portal.scripts.ImportTimelineData" : True
+    "org.mskcc.cbio.portal.scripts.ImportTimelineData" : True,
+    "org.mskcc.cbio.portal.scripts.ImportGenePanelProfileMap" : True
```

Comment:
I'll do this.  I'm replying without looking closer at the code, but I'm wondering why I haven't encountered these issue myself.   Perhaps its because I've been running cbioportalImporter.py directly?  

./core/src/main/scripts/importer/cbioportalImporter.py -jar ./scripts/target/scripts-*.jar -c import-study-data -m core/src/test/scripts/test_data/study_es_0/meta_gene_panel_matrix.txt -d core/src/test/scripts/test_data/study_es_0/data_gene_panel_matrix.txt

In any event, I'll fix the code as you suggest.

Thanks @fedde-s